evaluator/src/test/kotlin/com/xebia/functional/xef/evaluator/EitherOpsSpec.kt

```diff
@@ -0,0 +1,91 @@
+package com.xebia.functional.xef.evaluator
+
+import com.xebia.functional.xef.evaluator.EitherOps.toJson
+import com.xebia.functional.xef.evaluator.EitherOps.toJsonFile
+import com.xebia.functional.xef.evaluator.models.SuiteSpec
+import com.xebia.functional.xef.evaluator.utils.Generators.emptyString
+import com.xebia.functional.xef.evaluator.utils.Generators.simpleString
+import io.kotest.assertions.json.shouldBeJsonObject
+import io.kotest.assertions.json.shouldBeValidJson
+import io.kotest.assertions.json.shouldContainJsonKey
+import io.kotest.assertions.json.shouldEqualJson
+import io.kotest.common.runBlocking
+import io.kotest.matchers.shouldBe
+import io.kotest.property.arbitrary.next
+import kotlin.io.path.absolutePathString
+import kotlin.io.path.createTempDirectory
+import kotlin.io.path.listDirectoryEntries
+import kotlin.test.Test
+import kotlinx.coroutines.test.runTest
+
+class EitherOpsSpec {
+  private val validSuiteSpec = runBlocking {
+    SuiteSpec(simpleString.next()) {
+      outputDescription { simpleString.next() }
```

Comment:
Each time this test is executed, it runs once with random data, right? In that case, I think it would be better to use fixed values because, right now, can lead to flaky tests. Not a big deal though, just a preference.